oximeter/impl/src/schema/mod.rs

```diff
@@ -153,18 +159,58 @@ fn validate_timeseries_name(s: &str) -> Result<&str, MetricsError> {
     }
 }
 
+/// Text descriptions for the target and metric of a timeseries.
+#[derive(Clone, Debug, Default, Deserialize, JsonSchema, Serialize)]
```

Comment:
Does `Default` make sense here? That would just be an empty description, right?

Reply:
Correct, it's an empty description, and I think that does make sense. The intent was to support the sequence of changes where we start _adding_ these extra pieces of metadata to existing schema that don't currently have them. This type needs `Default` implemented so we can ask serde to create us a default on deserialization.

This is one of the areas I do appreciate extra eyes on though. I am added default values for deserialization to all the new fields of a schema:

- version (default = 1)
- description (default is empty, which you noted here)
- authz_scope (defaults to fleet)
- units (defaults to count)

I _think_ this is the right step. The plan is to add defaults for all these so that we can send them across serialization boundaries; add them into the CRDB and ClickHouse schema; and then start rewriting schema to use the TOML format where these will be provided with "real" values. But I need the defaults first, to support serialization.

Reply:
My previous comment isn't quite right. We don't actually need defaults for deserialization, so I'm going to remove this trait derivation.